src/Interpreters/GinFilter.h

```diff
@@ -72,45 +123,28 @@ class GinFilter
     void addRowRangeToGinFilter(UInt32 segment_id, UInt32 rowid_start, UInt32 rowid_end);
 
     /// Clear the content
-    void clear();
-
-    /// Check if the filter (built from query string) contains any rows in given filter by using
-    /// given postings list cache
-    bool contains(const GinFilter & filter, PostingsCacheForStore & cache_store, GinSearchMode mode = GinSearchMode::All) const;
-
-    /// Set the query string of the filter
-    void setQueryString(std::string_view query_string_)
+    void clear()
```

Comment:
Can we move the implementation back into the source code file? (just as a matter of principle, as little stuff as possible should be in headers).